webapp/packages/plugin-data-import/src/DataImportDialog/DataImportFileUploader.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { observer } from 'mobx-react-lite';
+
+import { Container, Group, InputFiles, s, Table, TableBody, TableColumnHeader, TableHeader, useS, useTranslate } from '@cloudbeaver/core-blocks';
+
+import { DataImportFileItem } from './DataImportFileItem';
+import styles from './DataImportFileUploader.m.css';
+import type { IDataImportDialogState } from './IDataImportDialogState';
+
+interface Props {
+  state: IDataImportDialogState;
+  onDelete: () => void;
+}
+
+export const DataImportFileUploader = observer<Props>(function DataImportFileUploader({ state, onDelete }) {
+  const translate = useTranslate();
+  const style = useS(styles);
+
+  function onUpload(value: FileList | null) {
+    if (value) {
+      state.file = value[0];
+    }
+  }
```

Comment:
please use changeHandler or handleChange or handleFileSelect (because this component not uploading anything anywhere)